pygame.py

```diff
@@ -0,0 +1,67 @@
+# Needed to use any and all python resources.
+import pygame
+import pygame_menu  #pip install pygame-menu
+from os import path
+from win32api import GetSystemMetrics 
+import yahtzee        
+
+# Defines common colors
+background_one = (255, 255, 255)  # white
+background_two = (0, 0, 0)  # black
+FPS = 30 # frames per second
+
+# Set the size of the window, variables can be used for positioning
+# within program.
+window_width = GetSystemMetrics((0))
+window_height = GetSystemMetrics((1))
+
+
+#create function to present text
+font_name = pygame.font.match_font('arial') 
+def text_draw(surface, text, size, x, y):
+    font = pygame.font.Font(font_name, size)
+    text_surface = font.render(text, True, background_two)
+    text_rect = text_surface.get_rect()
+    text_rect.midtop = (x, y)
+    surface.blit(text_surface, text_rect)
```

Comment:
I wonder if you also need to create a variable for the size if you created one for font_name?